packages/cli/README.md

```diff
@@ -15,47 +15,50 @@ $ npx @lionweb/cli
 
 produces information about which commands are available.
 
-## Extracting essential information from a serialization chunk
+This CLI utility does not perform any explicit validation apart from the file at the given path existing and being valid JSON.
+It does some implicit validation as it can error out on incorrect serializations.
+
+
+## Sorting a serialization chunk
 
-Run the following command to make "extractions" from a serialization chunk (e.g.):
+Run the following command to sort a serialization chunk (e.g.):
 
 ```shell
-npx @lionweb/cli extract ../artifacts/chunks/languages/lioncore.json
+npx @lionweb/cli sort ../artifacts/chunks/languages/lioncore.json
 ```
 
-This is meant as a way to inspect, reason about, and compare serialization because the format is rather verbose.
-These extractions are:
+Sorting a serialization chunk means the following:
 
--   A "sorted" JSON with:
-    -   all nodes sorted by ID,
-    -   for all nodes, their properties, containments, and references sorted by key (from the meta-pointer),
-    -   and all containments and references sorted by ID.
--   A "shortened" JSON where keys are used as key names.
--   If the serialization represents a language - i.e.: a LionCore model - then a textual version is generated as well.
+* All nodes sorted by ID.
+* For all nodes, their properties, containments, and references sorted by key (from the meta-pointer),
+* All containments and references sorted by ID.
+
+The sorting produces a serialization chunk that's also aligned on JSON-level with the specification.
 
-This CLI utility does not perform any explicit validation apart from the file at the given path existing and being valid JSON.
-It does some implicit validation as it can error out on incorrect serializations.
 
 ## Changelog
 
 ### 0.6.1
 
--   Change the `diagram` command to output PlantUML and Mermaid diagram files _per language_.
--   Add a validation commmand to validate JSON files as serialization chunks
+* Change the `diagram` command to output PlantUML and Mermaid diagram files _per language_.
+* Add a `validation` commmand to validate JSON files as serialization chunks.
+* Remove the shortening functionality from `extract`.
+* Rename the `extract` command &rarr; `sort`, and remove the textualization for chunks that are the serialization of languages.
+* Textualizing a serialization chunk of languages will use the LionCore/M3-specific syntax _unless_ the flag `--languagesAsRegular` is an argument.
 
 ### 0.6.0
 
--   Add a `repair` command that aligns a JSON serialization chunk with the specification.
--   Add a `textualize` command – that's optionally language-aware – to render a JSON serialization chunk as pure text.
+* Add a `repair` command that aligns a JSON serialization chunk with the specification.
```

Comment:
Can you elaborate what "aligns" means?

Reply:
Align in the sense of matching exactly with the order of the key-value pairs in the specification. This is also in the interest of introspectivity, predictability, etc.